package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "actions-wiki",
+  "private": true,
+  "version": "0.4.0",
+  "description": "",
+  "main": "lib/index.js",
+  "scripts": {
+    "build": "tsc",
+    "format": "prettier --write .",
```

Comment:
```diff
-    "format": "prettier --write .",
+    "format": "prettier --ignore-path .gitignore -w .",
```

This would make it so you don't need a .prettierignore since your .gitignore would double as your .prettier ignore.
Though then again this might conflict with the fact that there's minified JS in dist/ in source control 🤦‍♀️

Reply:
Yeah, prettifying built files is an annoyance that I'd like to avoid.